examples/index.html

```diff
@@ -178,8 +178,13 @@ <h4 class="modal-title">Your gif is ready</h4>
                 </div>
               </div>
               <div class="modal-footer">
-                <input type='number' value=1 id="js-download-gif-framerate" />
-                <button type="button" id="js-download-gif-framerate-btn" class="btn btn-default">Apply</button>
+                <div>
+                  <div>
+                  <p>Framerates</p>
```

Comment:
Can You put this on top of input element rather than the apply button right now. Also is Framerates the correct word in this context . Shouldn't it be Frame-Duration?

Reply:
Yea you're right, It should be called frameduration, I'll change it ASAP

Reply:
Done!